hbase-server/src/main/java/org/apache/hadoop/hbase/master/region/MasterRegionFlusherAndCompactor.java

```diff
@@ -263,6 +263,14 @@ void requestFlush() {
     }
   }
 
+  void resetChangesAfterLastFlush() {
+    changesAfterLastFlush.set(0);
+  }
+
+  void resetLastFlushTime() {
```

Comment:
Should name it recordLastFlushTime?

Reply:
Thanks @Apache9 for the review.
I have modified it in the latest commit.